Properties/Resources.resx

```diff
@@ -0,0 +1,247 @@
+{1} - The expected type of the parameter.
+{2} - The actual type of the parameter.</comment>
+  </data>
+  <data name="FileAlreadyExists" xml:space="preserve">
+    <value>The file already exists.</value>
+    <comment></comment>
+  </data>
+  <data name="FilePathAlreadyExists_file" xml:space="preserve">
+    <value>File {0} already exists.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="OutOfMemory" xml:space="preserve">
+    <value>libzip returned out of memory error.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamFailedConsistencyChecks" xml:space="preserve">
+    <value>The stream failed consistency checks.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamDoesNotExist" xml:space="preserve">
+    <value>Stream does not exist and file creation wasn't requested.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamIsNotAZip" xml:space="preserve">
+    <value>Stream is not a ZIP archive.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamCouldNotBeOpened" xml:space="preserve">
+    <value>Stream could not be opened.</value>
+    <comment></comment>
+  </data>
+  <data name="ErrorReadingStream" xml:space="preserve">
+    <value>Error occured while reading the Stream.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamDoesNotSupportSeeking" xml:space="preserve">
+    <value>Stream does not support seeking.</value>
+    <comment></comment>
+  </data>
+  <data name="UnexpectedLibZipError_error" xml:space="preserve">
+    <value>Unexpected libzip error: {0}.</value>
+    <comment>{0} - An error message from libzip.</comment>
+  </data>
+  <data name="FileFailedConsistencyChecks_file" xml:space="preserve">
+    <value>The file {0} failed consistency checks.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="FileDoesNotExist_file" xml:space="preserve">
+    <value>The file {0} does not exist and file creation wasn't requested.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="FileIsNotAZip_file" xml:space="preserve">
+    <value>The file {0} is not a ZIP archive.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="FileCouldNotBeOpened_file" xml:space="preserve">
+    <value>The file {0} could not be opened.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="ErrorReadingFile_file" xml:space="preserve">
+    <value>Error occured while reading the file {0}.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="FileDoesNotSupportSeeking_file" xml:space="preserve">
+    <value>The file {0} does not support seeking.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="DestinationMustNotBeEmpty" xml:space="preserve">
+    <value>Archive destination path must not be empty.</value>
+    <comment></comment>
+  </data>
+  <data name="UnknownError" xml:space="preserve">
+    <value>Unknown Error.</value>
```

Comment:
Maybe just to remove any ambiguity for the localization team about whether Error is capitalized for a special reason in this context:

```
<value>Unknown error</value>
```